ga4gh/gff3.py

```diff
@@ -134,25 +181,41 @@ def __linkToParent(self, feature, parentId):
             parentPart.children.add(feature)
 
     def finish(self):
-        """
-        finish loading the set, constructing the tree
-        """
+        "finish loading the set, constructing the tree"
```

Comment:
@macieksmuga, change this finish() to follow the server
conventions. Note to the python interpret both ".." and
"""..."""  are doc strings.  I don't follow the """ convention
in my code and I missed changing this one.

Maciek Smuga-Otto <notifications@github.com> writes:
> In ga4gh/gff3.py:
> 
> > @@ -134,25 +181,41 @@ def __linkToParent(self, feature, parentId):
> >              parentPart.children.add(feature)
> >
> >      def finish(self):
> > -        """
> > -        finish loading the set, constructing the tree
> > -        """
> > +        "finish loading the set, constructing the tree"
> 
> why not a docstring here?
> 
> —
> Reply to this email directly or view it on GitHub.*
>